engine/src/main/java/org/hibernate/search/annotations/IndexedEmbedded.java

```diff
@@ -34,9 +34,9 @@
 	String DEFAULT_NULL_TOKEN = "__DEFAULT_NULL_TOKEN__";
 
 	/**
-	 * Field name prefix, defaults to {@code propertyname.}.
+	 * Field name prefix.
 	 *
-	 * @return the field name prefix. Default to "."
+	 * @return the field name prefix. Default to ".", indicating that {@code propertyname.} should be used.
```

Comment:
Well, that's the word used elsewhere when describing the behavior of Hibernate Search.
See for instance:

```
	/**
	 * Default value for the {@link #indexNullAs} parameter. Indicates that {@code null} values should not be indexed.
	 */
	String DO_NOT_INDEX_NULL = "__DO_NOT_INDEX_NULL__";

	/**
	 * Value for the {@link #indexNullAs} parameter indicating that {@code null} values should be indexed using the null
	 * token given through the {@link org.hibernate.search.cfg.Environment#DEFAULT_NULL_TOKEN} configuration property.
	 * If no value is given for that property, the token {@code _null_} will be used.
	 */
	String DEFAULT_NULL_TOKEN = "__DEFAULT_NULL_TOKEN__";
```

```
	/**
	 * Returns {@code true}, if the id of the embedded object should be included into the index,
	 * {@code false} otherwise. The default is {@code false}.
	 *
	 * <p><b>Note</b><br>:
	 * If the id property is explicitly listed via {@link #includePaths()}, then the id is included even if this value
	 * is {@code false}.
	 * </p>
	 *
	 * @return Returns {@code true}, if the id of the embedded object should be included into the index,
	 * {@code false} otherwise.
	 */
	boolean includeEmbeddedObjectId() default false;
```

And so on.

Anyway, I'll change that.